### PR TITLE
Preserve proposal document title for proposal documents of submitted proposals

### LINKED ISSUE #4845: Proposal documents don't keep their title when submitted
There is an issue when submitting a proposal where the proposal document has a different title than the proposal. I would expect that the submitted proposal's document uses the same title as the proposal's document. The document title is reset to the submitted proposal title, however.

Steps to reproduce:
- Create a proposal
- Set a new title for the proposal's document which is different from the proposals title
- Submit the proposal
- Go to the submitted proposal and view its document

Expected outcome:
- The document of the submitted proposal should have the same title as the document of the proposal

Actual outcome:
- The document's title is set to the title of the proposal (see https://github.com/4teamwork/opengever.core/blob/1cff81b94320be2512a27841f1ea4c7d0f50f1c3/opengever/meeting/proposal.py#L312)

### PR DESCRIPTION
**Pull request by Rotonen**
Closes #4845

### CODE REVIEW

File: opengever/meeting/browser/createsubmittedproposal.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def __call__(self):
             submitted_proposal.sync_model(proposal_model=proposal)
 
             submitted_proposal.create_proposal_document(
+                title=proposal_oguid.resolve_object().get_proposal_document().title_or_id(),
```
Comment (deiferni) [inline opengever/meeting/browser/createsubmittedproposal.py]: We're attempting to keep the workload necessary to make the meeting stuff work across admin units low. Thus you should not get the proposal document here as that may be on another plone site.
IIRC the document and file data is provided in the request. Could you pick the title out of there (title should be identical to the filename)?

Comment (deiferni) [reply opengever/meeting/browser/createsubmittedproposal.py]: for the record as just discussed: the title could be added in https://github.com/4teamwork/opengever.core/blob/df27d21d1517041e2fdde8d6fe7809e6680daa93/opengever/meeting/command.py#L425-L428 and then retrieved from the request when creating the document.

**Comment by Rotonen** [comment]
Ready for rereview.

**Comment by Rotonen** [comment]
@deiferni nixed the reformats on anything I did not move or actually touch.

**Pull request merged**